core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/LazyHttpClient.java

```diff
@@ -64,56 +47,43 @@
     @GuardedBy("lock")
     private RuntimeException initException;
     @GuardedBy("lock")
-    private CloseableHttpClient httpClient;
-
-    static ApacheSender43 create() {
-        return new ApacheSender43();
-    }
-
-    private ApacheSender43() {
-    }
+    private CloseableHttpClient delegate;
 
-    @Override
-    public HttpResponse sendRequest(HttpUriRequest request) throws IOException, FriendlyException {
-        try {
-            return getHttpClient().execute(request);
-        } catch (SSLHandshakeException e) {
-            String completeUrl = "https://" + request.getURI().getHost();
-            throw SSLUtil.newSSLFriendlyException(completeUrl);
-        }
+    public static CloseableHttpClient getInstance() {
```

Comment:
can we return LazyHttpClient? I know LazyHttpClient is an instance of closeableHttpClient, Thought that will make it clear between delegate and instance.

Reply:
I think this is ok, reflects that callers don't need any special details of LazyHttpClient, only the interface